.gitlab-ci.yml

```diff
@@ -656,6 +656,7 @@ library:ci-fiat_crypto:
   needs:
   - build:edge+flambda
   - library:ci-coqprime
+  - library:ci-bedrock2
```

Comment:
possibly need to add it to the fiat_crypto_ocaml job too, although I never fully understood the rules for when we need something in need so we can try without if you want

Reply:
I think we don't; you need to add deps that must be installed for the job to run.  For most Coq jobs this is everything, but fiat_crypto_ocaml is special because it only invokes ocamlopt on already extracted files, and hence does not require any transitive dependencies to be installed.